iso15118/shared/utils.py

```diff
@@ -49,3 +56,10 @@ async def wait_for_tasks(
             task.result()
         except Exception as e:
             logger.exception(e)
+
+
+async def start_and_wait_for_coroutines(
+    await_coroutines: List[Coroutine], return_when=asyncio.FIRST_EXCEPTION
+):
+    tasks = start_tasks(await_coroutines)
+    await wait_for_tasks(tasks)
```

Comment:
return when is not passed to the wait_for_tasks method

Reply:
good point. I've added a commit to fix that...